_includes/manuals/3.16/1.2_release_notes.md

```diff
@@ -4,16 +4,222 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### User Interface & Experience
+- **All Hosts Page Redesign** - Continued modernization of the All Hosts page with enhanced functionality, eliminating AngularJS pages and improving user experience for managing thousands of hosts
+- **Job Invocations Detail Page** - Complete overhaul of job execution details with improved performance, better loading, and enhanced user interface
+
+#### Infrastructure & Platform Updates
+- **NodeJS 22 Upgrade** - Updated UI build stack to NodeJS 22, extending lifecycle support to April 2027
+- **PatternFly 5 Migration** - Modernization of UI components by upgrading from PatternFly 3 to PatternFly 5 in Foreman core
```

Comment:
We upgraded the Patternfly 4 components in core and plugins to use 5
we still use pf3

Reply:
@MariaAga How does this sound to you?
```suggestion
- **PatternFly 5 Migration** - Modernization of UI components by upgrading from PatternFly 4 to PatternFly 5 in Foreman core
```